mod-finance/examples/group_fiscal_year_summary_collection.sample

```diff
@@ -0,0 +1,19 @@
+{
+  "groupFiscalYearSummaries": [
+    {
+      "groupId": "f33ed99b-852a-4f90-9891-5efe0feab165",
+      "fiscalYearId": "ac2164c7-ba3d-1bc2-a12c-e35ceccbfaf2",
+      "allocated": 100000.00,
+      "available": 60000.00,
+      "unavailable": 40000.00
+    },
+    {
+      "groupId": "66f84ff8-68e1-4f10-8814-35bcd34c32a0",
+      "fiscalYearId": "a4ee40d4-5176-4b76-aa46-2e17e9161795",
```

Comment:
I think typically you'd be querying by fiscalYearId so these would be the same, but what you have here technically isn't impossible.  No need to change it, just some commentary.